oxide/wrap_libport_mgr_hw/shim.c

```diff
@@ -0,0 +1,72 @@
+void *stdout;
```

Comment:
Should we have a file-header comment about this shim file and what they are shimming/attached-to? 

I know you mentioned this in the PR.

> One item in particular that may need further explanation is oxide/wrap_libport_mgr_hw. The open source SDE comes with a pre-built library that was linked against a specific version of the linux glibc. This wrapper splits that library back into its original .o files, provides a shim that implements the needed glibc functions, and relinks the library for helios.